clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -48,6 +50,12 @@ const (
 	checkPrometheusReadyTimeout  = 15 * time.Minute
 	numK8sClients                = 1
 	nodeExporterPod              = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/exporters/node-exporter.yaml"
+	wmiExporterURL               = "https://github.com/martinlindhe/wmi_exporter/releases/download/v0.10.1/wmi_exporter-0.10.1-amd64.msi"
+	wmiExporterCollectors        = "process,cs"
```

Comment:
It would be nice if there was a way to specify this as part of the yaml config somewhere, instead of hard-coding it here. Not sure if there's an easy way to do that though.

Reply:
Yeah, would be great. Also this file is quite large already (close to 400 lines). Can we extract all the windows features to a separate file and just call something from here?

Reply:
Not difficult, moved to `windows_override.yaml`